migrations/sql/V2019.12.21.23.39__support_multiple_incident_types.sql

```diff
@@ -0,0 +1,46 @@
+ALTER TABLE mine_incident DROP CONSTRAINT IF EXISTS mine_incident_mine_incident_category_code_fkey;
+ALTER TABLE mine_incident DROP COLUMN IF EXISTS mine_incident_category_code;
+
+ALTER TABLE mine_incident_category ALTER COLUMN mine_incident_category_code TYPE varchar(3);
+
+ALTER TABLE mine_incident_category DROP COLUMN IF EXISTS display_order;
+ALTER TABLE mine_incident_category ADD COLUMN display_order integer NOT NULL DEFAULT 0;
+
+UPDATE mine_incident_category
+SET display_order = 0
+WHERE mine_incident_category_code = 'ENV';
+
+UPDATE mine_incident_category
+SET display_order = 1
+WHERE mine_incident_category_code = 'GTC';
+
+UPDATE mine_incident_category
+SET display_order = 2
+WHERE mine_incident_category_code = 'H&S';
+
+UPDATE mine_incident_category
+SET display_order = 3
```

Comment:
can we change the display orders to be 10, 20, 30, 40 so that we can add new records between in the future.